kili/services/export/format/kili/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""
+Functions to export a project to YOLOv4, v5 or v7 format
+"""
+
+from ...repository import SDKContentRepository
+from ...tools import fetch_assets
+from ..base import (
+    BaseExporterSelector,
+    ContentRepositoryParams,
+    ExportParams,
+    LoggerParams,
+)
+from .common import KiliExporter
+
+
+class KiliExporterSelector(BaseExporterSelector):
+    # pylint: disable=too-few-public-methods
+
+    """
+    Formatter to export to Kili
+    """
+
+    @staticmethod
+    def export_project(
+        kili,
+        export_params: ExportParams,
+        logger_params: LoggerParams,
+        content_repository_params: ContentRepositoryParams,
+    ) -> None:
+        """
+        Export a project to Kili format
+        """
+        logger = BaseExporterSelector.get_logger(logger_params.level)
+
+        logger.info("Fetching assets ...")
+        assets = fetch_assets(
```

Comment:
there is a lot that can be factored with the yolo exporter:
* export_project can belong to the base selector
* the process method can be an abstract method of the base selector...
* ... that is implemented in the Kili selector and the Yolo selector

Reply:
I factorized some of the code but I am not sure that it was what you had in mind